text/0000-atcompoundkey-error-path-optimization.md

```diff
@@ -0,0 +1,197 @@
+- Start Date: 2018-10-05
+- RFC PR: 
+- CppMicroServices Issue: https://github.com/CppMicroServices/CppMicroServices/issues/302
+
+# Improving the error path performance for AtCompoundkey API
+
+## Motivation
+
+Using AtCompoundKey with non-existent keys produces an exception. If the API is used extensively, the application performace is degraded as reported by a user. Users prefer not to pay for the cost of exception handling in case a compound key is not found in the AnyMap.
+
+## Description of the Problem
```

Comment:
What are the specific use cases under which this is a problem for the user? I don't see a description of the conditions which makes the error path a performance problem.